### PR TITLE
Docs/changelog initial

### PR DESCRIPTION
**Pull request by sapphire-arches**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
+## c1.1.0 Beethoven
+
+### Sound Engine
+
+- Added a version of Émilie Gillet's Mutable Instruments Clouds reverb model. Can be enabled via a new `REVERB MODE` sub-menu under the existing Reverb menu.
+- Fixed some bugs around the waveform Loop Lock feature which allowed setting invalid loop points.
+
+### User Interface
+
+- Added `PERFORMANCE VIEW`, accessible in Song View by pressing the Keyboard button. Allows quick control Song Global FX.
```
Comment (seangoodvibes) [inline CHANGELOG.md]: Nit: "Allows quick control of Song Global FX."

Add word "of"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,77 +1,77 @@
...
+- Changed default ModFX type is now DISABLED rather than FLANGER.
+
+### Kits
+
+- Drum randomization is no longer limited to only 10 sounds per folder.
+- Fixed several crashes related to drum randomization.
+
+### MIDI
+
+- Added `MIDI FOLLOW MODE` which causes MIDI data to be directed to the currently focused clip.
```
Comment (seangoodvibes) [inline CHANGELOG.md]: Not just the focused clip.

- In the Song View's (Session, Arranger, Performance) parameter mapped cc's will control the Song Global FX
- Notes, MPE and related CC's will be directed to the currently focused clip.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,77 +1,77 @@
...
+- Added MIDI feedback, so external MIDI controllers can be made aware of the state of the Deluge synth engine. Configurable via the new global `MIDI > MIDI-FOLLOW > FEEDBACK` menu.
+- Added Loopy Pro and TouchOsc templates for MIDI feedback.
```
Comment (seangoodvibes) [inline CHANGELOG.md]: These templates not just for MIDI feedback. They are for MIDI Follow and MIDI Feedback.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,77 +1,77 @@
...
+- Added a `MIDI LOOPBACK` mode, accessible in the SONG menu, which directs MIDI data from internal MIDI clips back to the Deluge input.
+- Added support for learning Program Change methods to most global commands.
+- Added "MPE collapse" on MIDI clips which converts MPE X/Y/Z to Pitch/Modwheel/Aftertouch CCs for use of MPE controllers with non-MPE aware synths. Configurable via the clip menu.
+- Added a new global setting, `MIDI > SELECT KIT ROW`, which causes MIDI notes sent to kits to be sent to the individual row. Useful for CC control of individual kit rows.
```
Comment (seangoodvibes) [inline CHANGELOG.md]: Doesn't cause MIDI notes for kits to be sent to the individual row. Rather it just selects the kit row (like if you were to select it by manually pressing on the audition pad for that row). It's useful for being able to control the parameters of that kit row using the gold knobs or midi follow cc's (when affect entire is disabled).

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,77 +1,77 @@
...
+- Fixed reversal of upstream MIDI ports (port 1 was upstream port 2, port 2 was upstream port 1).
+- Fixed a number of MPE channel matching and learning bugs.
+- Fixed crash when using an external controller to control ModFX.
```
Comment (seangoodvibes) [inline CHANGELOG.md:65]: Speaking of ModFX, should we mentioned that for Audio Clips we now allow Mod FX to be Disabled in Audio Clips, Song and Kit Affect Entire and are set to Disabled by default (they were set to Flanger by default previously and could not be disabled).

**Review by seangoodvibes — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (sapphire-arches) [reply CHANGELOG.md]: Rather than try to re-explain the logic, I just changed it to say "based on context"

**Review by seangoodvibes — APPROVED** [review]

**Pull request merged**